src/backend/EdFi.DataManagementService.Backend.Postgresql.Test.Integration/UpsertTests.cs

```diff
@@ -77,15 +34,15 @@ public static IGetRequest CreateGetRequest(Guid documentUuidGuid)
         return (
             new
             {
-                ResourceInfo = _resourceInfo,
+                ResourceInfo = ResourceInfo,
                 TraceId = new TraceId("123"),
                 DocumentUuid = new DocumentUuid(documentUuidGuid)
             }
         ).ActLike<IGetRequest>();
     }
 
     [TestFixture]
-    public class Given_an_upsert_of_a_new_document : UpsertTests
+    public class GivenAnUpsertOfANewDocument : UpsertTests
```

Comment:
I thought we intentionally chose a snake case naming convention for fixture classes?

Reply:
I think one of my tools must have done this for me. I don't remember doing it.